packages/fuels/src/cli/utils/getLatestFuelsVersion.ts

```diff
@@ -0,0 +1,25 @@
+import { checkAndLoadCache, saveToCache } from './fuelsVersionCache';
+
+export const getLatestFuelsVersion = async (): Promise<string | undefined> => {
```

Comment:
Could we potentially document the throwable conditions here?

Reply:
Not sure if I follow you, can you share some pseudo code? What do you mean by documenting the throwable conditions?